packages/contracts/src/EmailWalletCore.sol

```diff
@@ -394,6 +419,7 @@ contract EmailWalletCore is ReentrancyGuard, OwnableUpgradeable, UUPSUpgradeable
         require(amount > 0, "amount should be greater than 0");
         require(tokenAddr != address(0), "invalid token contract");
         require(emailAddrCommit != bytes32(0), "invalid emailAddrCommit");
+        // Sora's comment: Should this function replace `expiryTime` with `block.timestamp + unclaimedFundExpiryDuration` if the given value is zero?
```

Comment:
I think the check of ` require(expiryTime > block.timestamp, "invalid expiry time")` will always fail when `expiryTime=0` because `expiryTime` is replaced after this check.

Reply:
Oh right. I have fixed this.